app/services/my_anime_list_sync_service.rb

```diff
@@ -0,0 +1,161 @@
+class MyAnimeListSyncService
+  ATARASHII_API_HOST = 'https://hbv3-mal-api.herokuapp.com/2.1/'.freeze
+  MINE = '?mine=1'.freeze
+
+  attr_reader :library_entry, :method
+
+  def initialize(library_entry, method)
+    @library_entry = library_entry
+    @method = method
+  end
+
+  def execute_method
```

Comment:
Probably wanna check if `mal_media` is nil here, to handle stuff that's in our anime database but not MAL's